.gitbook/SUMMARY.md

```diff
@@ -1,27 +1,62 @@
 # Table of contents
 
-* [Introduction](README.md)
-
 ## Overview
 
-* [Panacea Ecosystem](overview/panacea-ecosystem.md)
-* [Roadmap](overview/roadmap.md)
-
-## Guide
-
-* [Getting started](guide/getting-started.md)
-* [Installation](guide/installation.md)
-* [Join the Network](guide/join-the-network.md)
-* [Deploy a Testnet](guide/deploy-a-testnet.md)
-* [Interaction with the network: CLI](guide/interaction-with-the-network-cli.md)
-* [Ledger Nano Support](guide/ledger-nano-support.md)
-* [Cosmovisor](guide/cosmovisor.md)
-
-## Specifications
-
-* [AOL](specifications/aol.md)
-* [DID](specifications/did.md)
-
-## Resources
-
-* [Validator FAQ](resources/validator-faq.md)
+* [Panacea Ecosystem](about-panacea/panacea-ecosystem.md)
+* [Roadmap](about-panacea/roadmap.md)
+
+## For Users
+
+* [Overview](users/overview.md)
+* Basic Concepts
+  * MED
+  * Transactions, Gas & Fees
+* Key Management
+  * CLI Keyring
+  * GUI Wallets
+* Data Exchange Protocol
+  * Concepts
+  * Consume data
+  * Provide data
+* Staking
+* Governance
+  * Overview
+  * Proposal Process
+  * Submit Proposals
+  * Vote to Proposals
+
+## For dApp Devs
+
+* [Overview](dapp-devs/overview.md)
+* Endpoints
+* Client Libraries
+
+## For Protocol Devs
+
+* [Overview](protocol-devs/overview.md)
+* Data Exchange Protocol Specs
+* Modules
+* Dev Guides
```

Comment:
How about adding a submenu for overall architecture of DEP?
Maybe you included it in the overview.

Reply:
i will make many submenus under the data exchange specs. It will contain the overall arch. The overview page is just for a headline of this section which is required by GitBook. So, it will make the overview page as simple as possible.